dexs/bitoro/index.ts

```diff
@@ -0,0 +1,47 @@
+import type { FetchResultVolume, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import { httpGet } from "../../utils/fetchURL";
+import { getUniqStartOfTodayTimestamp } from "../../helpers/getUniSubgraphVolume";
+import customBackfill from "../../helpers/customBackfill";
+
+// Bitoro volume API endpoint
+const bitoroApiVolumeEndpoint = "https://api.bitoro.network/btr/stats/volume"
+// March 25, 2024 UTC timestamp
+const bitoroLaunchDate = 1711324800
+// constructing request URL
+const makeRequest = (startTime: number, endTime: number) => bitoroApiVolumeEndpoint + `?start=${startTime}&end=${endTime}`
+
+const fetch = async (timestamp: number) => {
+    const startOfDayTimestamp = getUniqStartOfTodayTimestamp(new Date(timestamp * 1000)) + 86400
```

Comment:
I'm under the impression that 24hr volume is meant to be calculated for the current day, not the day before. getUniqStartOfTodayTimestamp gives the end of the prior day and we wanted start of current day. Is 24 hour volume meant to be for the day prior? I can see how this can be an issue with custom backfill

Reply:
is there tvl adapter ?

Reply:
No tvl adapter. Still waiting on endpoint.

Reply:
Any update?

Reply:
wait tvl adapter

Reply:
Can we not merge the volume adapter without a TVL adapter?

Reply:
yes

Reply:
Why is that the case?

Reply:
ppl will complaint why that protocol does not tvl but volume high is it fake volume and will delist

Reply:
Ok. We are working on this.